server/src/events/events.gateway.ts

```diff
@@ -18,12 +18,20 @@ export class EventsGateway implements OnGatewayConnection, OnGatewayDisconnect {
   handleDisconnect(client: any): void {}
 
   broadcast(data) {
-    this.server.clients.forEach((client) => client.send(data));
+    this.server.clients.forEach((client) => {
+      if (client.appId === data.appId) client.send(data.message);
```

Comment:
@gondar00 Get error on this line when compiling. Could you check?

Reply:
Yes, I have pushed the fix should work fine now 👍